transfermethodrepository/src/main/java/com/hyperwallet/android/ui/transfermethod/repository/TransferMethodConfigurationRepositoryImpl.java

```diff
@@ -184,6 +184,11 @@ public void refreshFields() {
         this.mTransferMethodType = mTransferMethodType;
     }
 
+    FieldMapKey(String mTransferMethodType) {
+        this.mCountry = null;
+        this.mCurrency = null;
+        this.mTransferMethodType = mTransferMethodType;
+    }
```

Comment:
why update this file?

Reply:
<img width="831" alt="Screenshot 2020-12-17 at 11 46 04 AM" src="https://user-images.githubusercontent.com/71114945/102451018-b3f13e00-405d-11eb-9c71-e1e42f94f33b.png">

FieldMapKey for type alone